runtime/bindings/python/src/pyopenvino/core/executable_network.cpp

```diff
@@ -21,26 +24,52 @@ void regclass_ExecutableNetwork(py::module m) {
         return InferRequestWrapper(self.create_infer_request(), self.inputs(), self.outputs());
     });
 
-    // cls.def("infer_new_request", [](ov::runtime::ExecutableNetwork& self, const py::dict& inputs) {
-    // TODO: implment after https://github.com/openvinotoolkit/openvino/pull/7962
-    // will be merged as a seperate ticket
-    // });
-
-    cls.def("export_model", &ov::runtime::ExecutableNetwork::export_model, py::arg("network_model"));
-
     cls.def(
-        "get_config",
-        [](ov::runtime::ExecutableNetwork& self, const std::string& name) -> py::handle {
-            return Common::parse_parameter(self.get_config(name));
+        "_infer_new_request",
+        [](ov::runtime::ExecutableNetwork& self, const py::dict& inputs) {
+            auto request = self.create_infer_request();
+            const auto key = inputs.begin()->first;
+            if (!inputs.empty()) {
+                if (py::isinstance<py::str>(key)) {
+                    auto inputs_map = Common::cast_to_tensor_name_map(inputs);
+                    for (auto&& input : inputs_map) {
+                        request.set_tensor(input.first, input.second);
+                    }
+                } else if (py::isinstance<py::int_>(key)) {
+                    auto inputs_map = Common::cast_to_tensor_index_map(inputs);
+                    for (auto&& input : inputs_map) {
+                        request.set_input_tensor(input.first, input.second);
+                    }
+                } else {
+                    throw py::type_error("Incompatible key type! Supported types are string and int.");
+                }
+            }
+
+            request.infer();
+
+            Containers::InferResults results;
+            for (const auto out : self.outputs()) {
+                results.push_back(request.get_tensor(out));
+            }
+            return results;
         },
-        py::arg("name"));
+        py::arg("inputs"));
 
-    cls.def(
```

Comment:
why have you commented this method?

Reply:
I was testing something and forgot to revert the changes, I will undo them in a second


> One more question which probably may help us -- have you tried to run tests on your machine from `openvino/runtime/bindgins/tests` directory? Did they pass?

I was locally runnig tests only for tests/test_inference_engine